lua/orgmode/objects/date.lua

```diff
@@ -11,25 +11,31 @@ local time_format = '%H:%M'
 
 ---@alias OrgDateSpan 'minute' | 'hour' | 'day' | 'week' | 'month' | 'year'
 
----@class OrgDate
----@field type string
+---@class OsTime
```

Comment:
This one does not have any fields. Since `os.time()` always returns a timestamp, we can probably just create an alias:

```suggestion
---@alias OsTime number
```

Reply:
Done.